cypress/e2e/view_errors/item_chart_fails_to_render.js

```diff
@@ -41,7 +41,7 @@ Given('I open a dashboard with a chart that will fail', () => {
 When(
     'I apply a {string} filter of type {string}',
     (dimensionType, filterName) => {
-        cy.contains('Add filter').click()
+        cy.contains('Filter').click()
```

Comment:
Do you think this is specific enough to always result in the correct element?

Reply:
I suppose it could break at some point. But I'd be in favour of keeping it until that happens. I like using text-based selectors for e2e tests where possible. In theory, we could change the implementation a lot but as long as the button label doesn't change (as it did this time around) the e2e test could still pass....

I could change it to a data-test selector, but would also have to add the data-test prop to the local `DropdownButton` component (which is not a big deal).

Reply:
I was thinking rather you could do something like:

`cy.get('button').containsExact('Filter').click()`. 

You can get `containsExact` from DV. Not sure it's in Dashboard